src/sql/workbench/services/accountManagement/browser/media/accountListRenderer.css

```diff
@@ -18,11 +18,11 @@
 	font-size: 15px;
 }
 
-.list-row.account-picker-list .label .display-name {
+.list-row.account-picker-list .label .display-name, .list-row.tenant-picker-list .label .display-name {
 	font-size: 13px;
 }
 
-.list-row.account-picker-list .label .content {
+.list-row.account-picker-list .label .content, .list-row.tenant-picker-list .label .content{
```

Comment:
The account logo below should almost definitely be scoped down.

Reply:
I don't want to change existing CSS :/

Reply:
Meh, not unreasonable but also I'd rather we fix this stuff when we see it and just deal with any consequences. Just doing a search for that class being used should be enough to tell you if changing it would affect anything else. But that's fine if you'd rather not risk it.